drivers/soundwire/sysfs_slave.c

```diff
@@ -212,3 +216,56 @@ int sdw_slave_sysfs_init(struct sdw_slave *slave)
 
 	return 0;
 }
+
+static const char *const slave_status[SDW_SLAVE_STATUS_NUM] = {
```

Comment:
in principle you don't need `SDW_SLAVE_STATUS_NUM` here, you could just do `slave_status[]`, then you wouldn't need to add `SDW_SLAVE_STATUS_NUM` at all.

Reply:
true, thanks for the suggestion.